handbook/digital-experience/digital-experience.rituals.yml

```diff
@@ -18,6 +18,13 @@
   description: "Run through the entire website in `?utm_content=clear` mode and build a fresh outline of the headings to make sure they all still make sense."
   moreInfoUrl: ""
   dri: "mike-j-thomas"
+- 
+  task: "Check brand fronts are up to date"
+  startedOn: "2024-08-01"
+  frequency: "Quarterly"
+  description: "Check all brand fronts for consistancy and update as needed with the current product pitch and graphics."
+  moreInfoUrl: "https://fleetdm.com/handbook/digital-experience#update-a-company-brand-front"
+  dri: "mike-j-thomas"  
```

Comment:
I think we need to go through this once with me and you before handing to Savannah.